### ISSUE TITLE
Ability to pass arguments to r2pipe script

### ISSUE THREAD

**Issue by XVilka** [radare/radare2#4721]
To be able pass arguments too when script is called using "." command:
```
[0x0000000]> . some_script.py argument1 argument2
```

**Comment by radare** [comment]
You dont want to run r2pipe scripts like this

Use #!pipe python script.py args

**Comment by radare** [comment]
Oh well this is a shortcut to do the same.. Should be done here too

**Comment by NishantPuri99** [comment]
I have a fair hold on Python, can I perhaps try this out? Thanks

**Comment by trufae** [comment]
I dont think it makes sense to pass arguments when using the . Command because its used to interpret a file.

when using #!pipe python foo.py foo bar it is possible to pass arguments already.

the other way we can use the % command to set the environment for example
